superset/assets/src/dashboard/components/HeaderActionsDropdown.jsx

```diff
@@ -108,6 +109,34 @@ class HeaderActionsDropdown extends React.PureComponent {
     this.props.startPeriodicRender(refreshInterval * 1000);
   }
 
+  downloadChartImage(ev){
```

Comment:
it looks like you haven't run the linter on your changes. you can run `npm run lint` and `npm run lint-fix` to clean this up